tests/tox_env/python/test_python_runner.py

```diff
@@ -95,7 +95,16 @@ def test_journal_package_dir(tmp_path: Path) -> None:
 
 def test_package_temp_dir_view(tox_project: ToxProjectCreator, demo_pkg_inline: Path) -> None:
     project = tox_project({"tox.ini": "[testenv]\npackage=wheel"})
-    result = project.run("r", "-vv", "-e", "py", "--root", str(demo_pkg_inline))
+    result = project.run(
```

Comment:
This seems an unrelated test, why did we change this?

Reply:
@kdestin may be able to speak to this better than I

Reply:
This test seems to have depended on the bug we're fixing. Adding `"--workdir", str(project.path / ".tox")` preserves the expected state of the tox invocation.